certification/internal/policy/operator/validate_operator_bundle.go

```diff
@@ -58,6 +77,78 @@ func (p ValidateOperatorBundleCheck) validate(report *cli.OperatorSdkBundleValid
 	return report.Passed, nil
 }
 
+func isTarget49OrGreater(ocpLabelIndex string) bool {
+	semVerOCPV1beta1Unsupported, _ := semver.ParseTolerant(ocpVerV1beta1Unsupported)
+	// the OCP range informed cannot allow carry on to OCP 4.9+
+	beginsEqual := strings.HasPrefix(ocpLabelIndex, "=")
+	// It means that the OCP label is =OCP version
+	if beginsEqual {
+		version := cleanStringToGetTheVersionToParse(strings.Split(ocpLabelIndex, "=")[1])
+		verParsed, err := semver.ParseTolerant(version)
+		if err != nil {
+			log.Errorf("unable to parse the value (%s) on (%s)", version, ocpLabelIndex)
+			return false
```

Comment:
What is the expected behavior if the check cannot parse the version in the annotation file? Does the check proceed with a ocp version <4.9? If so, it would be good to log a debug message in the Validate() function to log the ocp version that the check is running with.